packages/flame/lib/src/assets/images.dart

```diff
@@ -10,12 +10,18 @@ import 'package:flutter/services.dart';
 import '../flame.dart';
 
 class Images {
-  Images({this.prefix = 'assets/images/'})
-      : assert(prefix.isEmpty || prefix.endsWith('/'));
+  Images({String prefix = 'assets/images/'})
+      : assert(prefix.endsWith('/')),
+        _prefix = prefix;
```

Comment:
if you use the setter here you can have the assert on only one place?

Reply:
unfortunately dart don't allow that :(